src/sentry/replays/usecases/ingest/issue_creation.py

```diff
@@ -3,36 +3,64 @@
 
 from sentry.issues.grouptype import ReplayRageClickType
 from sentry.issues.issue_occurrence import IssueEvidence
+from sentry.models.project import Project
+from sentry.replays.query import query_replay_instance
 from sentry.replays.usecases.ingest.events import SentryEvent
 from sentry.replays.usecases.issue import new_issue_occurrence
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
 from sentry.utils import metrics
 
 logger = logging.getLogger()
 
+RAGE_CLICK_TITLE = "Suspected Rage Click"
+RAGE_CLICK_LEVEL = "warning"
 
+
+@instrumented_task(
+    name="sentry.replays.usecases.ingest.issue_creation.report_rage_click_issue",
+    queue="replays.ingest_replay",
+    default_retry_delay=5,
+    max_retries=5,
+    silo_mode=SiloMode.REGION,
+)
 def report_rage_click_issue(project_id: int, replay_id: str, event: SentryEvent):
     metrics.incr("replay.rage_click_issue_creation")
-
     payload = event["data"]["payload"]
 
+    project = Project.objects.get(id=project_id)
```

Comment:
You can remove this query if you pass `organization` as None to the querying function.  If we want to include the tenant-id we can pass the organization-id to the task and update the querying function to accept the id.

Reply:
mind if i do this in a follow up? i'll have to refactor the `query_replay_instance` function in general for this.